google/cloud/bigquery/schema.py

```diff
@@ -152,10 +136,10 @@ def from_api_repr(cls, api_repr: dict) -> "SchemaField":
         mode = api_repr.get("mode", "NULLABLE")
         description = api_repr.get("description", _DEFAULT_VALUE)
         fields = api_repr.get("fields", ())
+        policy_tags = api_repr.get("policyTags", _DEFAULT_VALUE)
 
-        policy_tags = cls._determine_policy_tags(
-            field_type, PolicyTagList.from_api_repr(api_repr.get("policyTags"))
-        )
+        if policy_tags is not None and policy_tags is not _DEFAULT_VALUE:
+            policy_tags = PolicyTagList.from_api_repr(api_repr.get("policyTags"))
```

Comment:
It seems we can just pass `policy_tags` as an argument directly, because we already checked that `"policyTags"`  key exists in `api_repr` (it's not `_DEFAULT_VALUE`) and is not `None`.

Reply:
Good call, done.

Also, I confused myself as to why I had `_DEFAULT_VALUE` in the first place, so I added some assertions in tests/unit/test_schema.py to make sure we aren't including the `policyTags` key when it's not present in the original API representation.